user_guide_src/source/outgoing/response.rst

```diff
@@ -481,7 +481,7 @@ The methods provided by the parent class that are available are:
         .. literalinclude:: response/023.php
 
         Only the ``name`` and ``value`` are required. To delete a cookie set it with the
-        ``expire`` blank.
+        ``value`` blank.
```

Comment:
Is this correct?

Reply:
Yes. 

> Only the **name** and **value** are required. To delete a cookie set it with the **expire** blank.

This is strange. What about the value?

> Note
> But if the value is set to empty string and the expire is set to 0, the cookie will be deleted.
https://codeigniter4.github.io/CodeIgniter4/outgoing/response.html#CodeIgniter\HTTP\Response::setCookie

If you set the value to empty string, the cookie will be deleted.
```php
$this->response->setCookie('test', '');
```

The cookie header will be:
```
Set-Cookie: test=deleted; expires=Thu, 01-Jan-1970 00:00:01 GMT; Max-Age=0; path=/; HttpOnly; SameSite=Lax
```

I've confirmed that the cookie was deleted on Firefox.

Reply:
Got it! Thanks for elaborating.